routing/payment_lifecycle.go

```diff
@@ -218,6 +218,9 @@ func (p *paymentLifecycle) createNewPaymentAttempt() (lnwire.ShortChannelID,
 		p.payment, uint32(p.currentHeight), p.finalCLTVDelta,
 	)
 	if err != nil {
+		log.Errorf("Failed to find route for payment %x: %v",
+			p.payment.PaymentHash, err)
```

Comment:
I think we should reserve `Error` for internal errors. This would be just `Info` I guess, not even warning?

Reply:
`warn` is a nice middle ground maybe?